libexec/setzer/setzer-price-xtzusd

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -e
+pair=xtzusd
+sources=(
+  binance
+  coinbase
+  kraken
+  okex
+  bitfinex
+  bitfinex
```

Comment:
bitfinex listed twice, hitbtc is missing

Reply:
fixed